dpnp/backend/extensions/elementwise_functions/common.hpp

```diff
@@ -0,0 +1,480 @@
+//*****************************************************************************
+// Copyright (c) 2025, Intel Corporation
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// maxification, are permitted provided that the following conditions are met:
```

Comment:
```suggestion
// modification, are permitted provided that the following conditions are met:
```

Probably a typo here and `float_power.cpp`, `fmax.cpp`, `fmin.cpp`, `gcd.cpp`, `heaviside.cpp`, `lcm.cpp`, `ldexp.cpp`, `logaddexp2.cpp`